programming_examples/CMakeLists.txt

```diff
@@ -138,3 +138,10 @@ add_lit_testsuite(check-reference-designs "Running the aie reference designs"
   ARGS "-sv --timeout 600"
 )
 set_target_properties(check-reference-designs PROPERTIES FOLDER "Tutorials")
+
+# Add programming_examples/utils to installed project.
+declare_mlir_python_sources(AIEPythonSources.ProgrammingExampleUtils
+  ADD_TO_PARENT AIEPythonSources
+  SOURCES
+    utils/parse_trace.py
+)
```

Comment:
Should it go in `aie/utils` if it's generally useful, instead of living in `programming_examples`?

Reply:
Feedbacks, comments and reviews are greatly appreciated. I haven't received any objections on this change so far, so I think I will land it some time tomorrow, if I do not receive any objections by then.